tracing-observation/src/main/java/com/couchbase/client/tracing/observation/ObservationRequestSpan.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2023 Couchbase, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.couchbase.client.tracing.observation;
+
+import java.time.Instant;
+
+import com.couchbase.client.core.cnc.RequestSpan;
+import com.couchbase.client.core.msg.RequestContext;
+import io.micrometer.observation.Observation;
+
+import static com.couchbase.client.core.util.Validators.notNull;
+
+/**
+ * Wraps a Micrometer Observation, ready to be passed in into options for each operation into the SDK as a parent.
+ *
+ * @author Marcin Grzejszczak
```

Comment:
Hi! I'm one of Graham's colleagues. As a team, we've been gradually removing Javadoc author tags from our codebase, since the version control history provides a more useful record of authorship. That said, I'll echo Graham's sentiment that your contribution is appreciated, and we'll be sure to credit you in the release notes unless you'd rather we didn't. :-)

Reply:
Lol😀 of course I'll remove the author tag, that's not a problem - we do it in micrometer and spring that's why I did here 🤷‍♂️

Reply:
Fixed!